evap/results/exporters.py

```diff
@@ -120,6 +120,14 @@ def export(self, response, course_types_list, include_not_enough_voters=False, i
             for evaluation, results in evaluations_with_results:
                 writec(self, evaluation.full_name, "evaluation")
 
+            writen(self, _("Course Degree"), "bold")
+            for evaluation, results in evaluations_with_results:
+                writec(self, ", ".join([d.name for d in evaluation.course.degrees.all()]), "border_left_right")
```

Comment:
please put every degree in a separate line (using `\n`). otherwise they won't be readable in the small columns in the exported file.